cypress/tests/ui/toolbar/global-toolbar.cy.js

```diff
@@ -1 +1,160 @@
 // All E2E Tests Related to global-toolbar goes here.
+
+describe('Global Toolbar', () => {
+  it('verifies that users can access the flowchart page through the flowchart icon, when in the experiment tracking view. #TC-1', function () {
+    cy.visit('/experiment-tracking');
+    cy.get('[data-test="View your pipeline"]').click();
+    cy.location('pathname').should('eq', '/');
+
+    // should exist
+    cy.get('.pipeline-wrapper').should('exist');
+
+    // should not exist
+    cy.get('.details__tabs').should('not.exist');
+  });
+
+  it('verifies that users can access the experiment tracking page through the experiment tracking button, when in the flowchart view. #TC-2', function () {
+    cy.get('[data-test="View your experiments"]').click();
+    cy.location('pathname').should('eq', '/experiment-tracking');
+
+    // should exist
+    cy.get('.details__tabs').should('exist');
+
+    // should not exist
+    cy.get('.pipeline-wrapper').should('not.exist');
+  });
+
+  it('verifies that users can change the theme from light to dark theme, or dark to light theme. #TC-3', function () {
+    // Alias
+    cy.get('[data-test="Toggle Theme"]').as('toggleTheme');
+
+    // Assert before action
+    cy.get('.kui-theme--dark').should('exist');
+    cy.__checkForAriaLabel__('@toggleTheme', 'Change to light theme');
+
+    // Action
+    cy.get('@toggleTheme').click();
+
+    // Assert after action
+    cy.get('.kui-theme--light').should('exist');
+    cy.__checkForAriaLabel__('@toggleTheme', 'Change to dark theme');
+  });
+
+  it('verifies that users can access the settings panel with the settings button. #TC-4', function () {
+    cy.get('[data-test="Change the settings flags"]').click();
+    cy.get('.pipeline-settings-modal > [role="dialog"]')
+      .should('be.visible')
+      .then(($dialog) => {
+        cy.wrap($dialog)
+          .should('have.attr', 'class')
+          .and('contains', 'modal--visible');
+
+        cy.wrap($dialog).within(() => {
+          cy.get('.modal__title').should('have.text', 'Settings');
+        });
+      });
+  });
+
+  describe('Settings Panel', () => {
+    it('verifies that users can change the display of the names of their kedro nodes. #TC-5', function () {
```

Comment:
add Tynan mentioned. I would test for a complex node name with lot of underscores 

reporting.cancellation_policy_breakdown <-> Cancellation Policy Breakdown. 

and also in the flowchart + metadata panel